NEWS.md

```diff
@@ -2,20 +2,36 @@ This file contains a summary of important user-visible changes.
 
 **New Features**
 
+cvc5 1.0.8
+==========
+Note: This is a pre-release version for the upcoming version 1.1.0.
+
+- API: Added argument `fresh` to `Solver::declareSort` analogous to
+  `Solver::declareFun`.The option `--fresh-declarations` determines whether the
+  parser constructs fresh sorts for each declaration (true by default, which
+  matches the previous behavior).
+
+cvc5 1.0.7
+==========
+
 - API: C++ enums are now enum classes
```

Comment:
This shouldn't have been in 1.0.7, was it really?

Reply:
yes, you are right, fixed.